ulib/FStar.Ghost.fsti

```diff
@@ -45,6 +45,7 @@ module FStar.Ghost
 
 (** [erased t] is the computationally irrelevant counterpart of [t] *)
 [@@ erasable]
+new
```

Comment:
Can you please comment on this change, just curious what was the trigger.

Reply:
Not sure why this is still in this PR. I already pushed it separately to master:
https://github.com/FStarLang/FStar/commit/08d80b839c67d83d001c0ed5ac62ef17e68da90f

Reply:
the issue is that `let test (x:erased int) (y:int) = x == y` would succeed phase 1 and fail in phase 2. Marking `erased` as new (which it is) ensures that it also fails phase 1.